src/coreclr/vm/callingconvention.h

```diff
@@ -1223,6 +1223,9 @@ int ArgIteratorTemplate<ARGITERATOR_BASE>::GetNextOffset()
         m_idxGenReg = numRegistersUsed;
         m_ofsStack = 0;
         m_idxFPReg = 0;
+#elif defined(TARGET_WASM)
+        // we put everything on the stack, we don't have registers
```

Comment:
This looks like we're implicitly settling on a wasm calling convention here, we might want to mark every spot where these decisions are being baked in with a comment like WASM-FIXME or WASM-TODO so we can find them all easily later